packages/client/src/service/txpool.ts

```diff
@@ -34,6 +35,8 @@ const MIN_GAS_PRICE = BigInt(100000000) // .1 GWei
 const TX_MAX_DATA_SIZE = 128 * 1024 // 128KB
 const MAX_POOL_SIZE = 5000
 const MAX_TXS_PER_ACCOUNT = 100
+// keep for 1 epoch deep to handle reorgs
+const BLOBS_AND_PROOFS_CACHE_LENGTH = 6 * 32 * 1
```

Comment:
The comment seems to imply we cache blobs based upon one epoch, but this is not the case, we cache the txpool blob pool based on 6 (max blobs per block) times 32 (epoch length) transactions. This can thus include both pending or included transactions (in the future).

Also slight nit: maybe define (if we don't have any) 32 as epoch length, and pull the `6` from max blobs per block param?

Reply:
updated on similar lines